### PR TITLE
update ng-interactive experiment path handling

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Perform

1. an update ng-interactive experiment path handling (resolves inconsistent path conventions)
2. the refactoring suggested by @jfsoul

### CODE REVIEW

File: applications/app/services/ApplicationsSpecial2020Election.scala [hunk 1]
```diff
@@ -25,18 +37,12 @@ object ApplicationsSpecial2020Election {
...
-          // s"<link rel=\"amphtml\" href=\"https://amp.theguardian.com/${path}\">",
-
-          // Going for another solution then
-          Array("<link rel=\"amphtml\" href=\"https://amp.theguardian.com/", path, "\">").mkString(""),
-        )
-      }
-      case _ => Html("")
+    if (ApplicationsSpecial2020Election.pathIsSpecialHanding(path)) {
+      Html(
+        s"""<link rel="amphtml" href="https://amp.theguardian.com/${path}">""",
```
Comment (gtrufitt) [inline applications/app/services/ApplicationsSpecial2020Election.scala:42]: This won't be double slashed now will it? `/${path}` path being `"/worl...`?

Comment (shtukas) [reply applications/app/services/ApplicationsSpecial2020Election.scala:38]: Nope. I accidentally pointed that out in the comment :)
```
when called from `ApplicationsDotcomRenderingInterface.getRenderingTier` it comes without starting slash
```

Comment (shtukas) [reply applications/app/services/ApplicationsSpecial2020Election.scala:38]: Oh! Jezz. Yes. Let me correct that.

**Comment by shtukas** [comment]
Thanks for spotting that problem @gtrufitt 
```
ApplicationsDotcomRenderingInterface.getRenderingTier
path: world/ng-interactive/2020/oct/29/covid-vaccine-tracker-when-will-a-coronavirus-vaccine-be-ready

request.path: /world/ng-interactive/2020/oct/29/covid-vaccine-tracker-when-will-a-coronavirus-vaccine-be-ready
println(s"request.path: ${request.path}")
```

**Comment by shtukas** [comment]
Correct: 
```
<link rel="amphtml" href="https://amp.theguardian.com/world/ng-interactive/2020/oct/29/covid-vaccine-tracker-when-will-a-coronavirus-vaccine-be-ready">
```

**Pull request merged**